include/libopencm3/stm32/h7/rcc.h

```diff
@@ -158,8 +180,133 @@ LGPL License Terms @ref lgpl_license
 #define RCC_CFGR_SW_PLL1          0x3
 /**@}*/
 
+/** @defgroup rcc_d1cfgr_values RCC_D1CFGR Values
+ * @ingroup rcc_registers
+ * @{*/
+#define RCC_D1CFGR_D1CPRE_BYP       0x0
+#define RCC_D1CFGR_D1CPRE_DIV2      0x8
+#define RCC_D1CFGR_D1CPRE_DIV4      0x9
+#define RCC_D1CFGR_D1CPRE_DIV8      0xA
+#define RCC_D1CFGR_D1CPRE_DIV16     0xB
+#define RCC_D1CFGR_D1CPRE_DIV64     0xC
+#define RCC_D1CFGR_D1CPRE_DIV128    0xD
+#define RCC_D1CFGR_D1CPRE_DIV256    0xE
+#define RCC_D1CFGR_D1CPRE_DIV512    0xF
+#define RCC_D1CFGR_D1PPRE_BYP       0x0
+#define RCC_D1CFGR_D1PPRE_DIV2      0x4
+#define RCC_D1CFGR_D1PPRE_DIV4      0x5
+#define RCC_D1CFGR_D1PPRE_DIV8      0x6
+#define RCC_D1CFGR_D1PPRE_DIV16     0x7
+#define RCC_D1CFGR_D1HPRE_BYP       0x0
+#define RCC_D1CFGR_D1HPRE_DIV2      0x8
+#define RCC_D1CFGR_D1HPRE_DIV4      0x9
+#define RCC_D1CFGR_D1HPRE_DIV8      0xA
+#define RCC_D1CFGR_D1HPRE_DIV16     0xB
+#define RCC_D1CFGR_D1HPRE_DIV64     0xC
+#define RCC_D1CFGR_D1HPRE_DIV128    0xD
+#define RCC_D1CFGR_D1HPRE_DIV256    0xE
+#define RCC_D1CFGR_D1HPRE_DIV512    0xF
+
+#define RCC_D1CFGR_D1CPRE_SHIFT     8
+#define RCC_D1CFGR_D1PPRE_SHIFT     4
+#define RCC_D1CFGR_RSVD_BITMASK     (0xfffff080)
```

Comment:
unless you've got evidence of registers where "keeping them at their reset value" is _required_, we've never done this before, and I don't really want to start doing it now.  In every other case so far, writing them to zero has been perfectly fine, and far less work to maintain when they add/document bits.

Reply:
That's fine...I'm happy to get rid of these, I was just very pedantically following the user manual :-)